controller/src/beerocks/master/db/db.cpp

```diff
@@ -4018,3 +4019,22 @@ void db::set_prplmesh(const sMacAddr &mac)
     }
     get_node(mac)->is_prplmesh = true;
 }
+
+bool db::update_client_entry_in_persistent_db(
+    const sMacAddr &mac, std::unordered_map<std::string, std::string> values_map)
+{
+    auto db_entry = client_db_entry_from_mac(mac);
+
+    if (!bpl::db_has_entry(type_to_string(beerocks::eType::TYPE_CLIENT), db_entry)) {
```

Comment:
If you convert the type to string more than once, it's probably better to use a variable.

Reply:
fixed on all occurences